### ISSUE TITLE
Decide on External Outreach

### ISSUE THREAD

**Issue by chris-remus** [ethereum/ens#129]

**Comment by chris-remus** [comment]
Partial List - 

- MEW
- Metamask
- Parity

**Comment by chris-remus** [comment]
@Arachnid and @alexvandesande Any additional groups to contact? Please let me know if they're any you'd like me to contact.

**Comment by chris-remus** [comment]
@Arachnid and @alexvandesande Have either of you contacted any of these external parties? If so, is there anything to follow-up on? If not, should we still plan to?

**Comment by chris-remus** [comment]
Parity (Alex) - will work w/ENS, pending

Metamask (Nick) - working on Ropsten, will work on Mainnet when we launch (done)

MEW (Nick) - pending

**Comment by yuanfeiz** [comment]
I can outreach to the most popular wallet in China: https://token.im/
Let me know if anything I can help

**Comment by Arachnid** [comment]
@yuanfeiz Please do! Guidance for implementers is available [here](http://docs.ens.domains/en/latest/implementers.html#interacting-with-ens-offchain) and resolving names is described [here](http://docs.ens.domains/en/latest/userguide.html#resolving-ens-names); that should be everything they need to get started.

**Comment by yuanfeiz** [comment]
Ok, I'll also try to talk with the exchanges to see if they can also put the name besides the deposit address.

**Comment by chris-remus** [comment]
@yuanfeiz Did you have a chance to connect with token.im?

**Comment by yuanfeiz** [comment]
@chris-remus Yeah, already did. They had launched a mobile version ENS registrar, and the integration with ENS is on the way. Expecting to release in the next version.

**Comment by Arachnid** [comment]
@yuanfeiz Thanks!
